pyproject.toml

```diff
@@ -22,7 +22,7 @@ dependencies = [
     "astropy>=5.3",
     "BayesicFitting>=3.0.1",
     "crds>=11.17.14",
-    "drizzle>=1.15.0",
+    "drizzle @ git+https://github.com/mcara/drizzle.git@redesign-api",
```

Comment:
remember to change to @main before merge

Reply:
so changed